bothub/common/tasks.py

```diff
@@ -478,3 +482,46 @@ def migrate_repository(repository_version, auth_token, language, name_classifier
     instance.auth_token = auth_token
     instance.language = language
     return instance.start_migrate()
+
+
+@app.task(name="intent_suggestions")
+def intent_suggestions(intent_id, authorization_token):  # pragma: no cover
+    intent = RepositoryIntent.objects.get(pk=intent_id)
+    language = intent.repository_version.version_languages.first().language
+    try:
+        dataset = {}
+        if intent:
+            if language in settings.SUGGESTION_LANGUAGES:
+                if cache.get(intent.text):
+                    dataset[intent.text] = (
+                        cache.get(intent.text)
+                        .strip("][")
+                        .replace("'", "")
+                        .split(", ")
+                    )
+                else:
+                    data = {
+                        "intent": intent.text,
+                        "language": language,
+                        "n_sentences_to_generate": settings.N_SENTENCES_TO_GENERATE,
+                        "repository_version": intent.repository_version_id
+                    }
+                    suggestions = request_nlp(
+                        authorization_token, None, "intent_sentence_suggestion", data
+                    )
+                    random.shuffle(suggestions["suggested_sentences"])
+                    if suggestions["suggested_sentences"]:
+                        dataset[intent.text] = suggestions["suggested_sentences"][:10]
+                        cache.set(intent.text, str(dataset[intent.text]), timeout=TIMEOUT)
```

Comment:
instead of adding a global TIMEOUT variable, directly place the variable you already have